iosApp/Survey/Sources/Presentation/Modules/Login/LoginView.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  ContentView.swift
+//  Survey
+//
+//  Created by Bliss on 14/9/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct LoginView: View {
+
+    @State private var email: String = ""
+    @State private var password: String = ""
+
+    var body: some View {
+        ZStack {
+            ZStack {}
```

Comment:
Why don't you use a `Image` instead?

Reply:
@suho I had issues with smaller devices, push the fix now in https://github.com/blyscuit/kmm-swiftUI-IC/pull/50/commits/687a070ee07182f962c08d8259b06781681ea6a8 🙏